playbooks/roles/jumpbox/files/mysql_db_backup.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
+
+# Copyright (c) Microsoft Corporation. All Rights Reserved.
+# Licensed under the MIT license. See LICENSE file on the project webpage for details.
+
+# General Variables (mysql)
+MYSQL_ADDRESS="10.0.0.17"
```

Comment:
Suggest: Might not want to hard-code the IP address in case our address scheme changes in the future.